### PR TITLE
Tabs: Java examples

### PR DESCRIPTION
**Pull request by anezthes**
**Note**:
- Badges (Java) example isn't working. Tried the usual approach:

```
@CssImport(value = "tabs-badges.css", include = "lumo-badge")
@JsModule("@vaadin/vaadin-lumo-styles/badge")
```

If this needs to be hacked, it doesn't offer much value to devs. Alternatively, we move it to Cookbook.

### CODE REVIEW

File: frontend/demo/component/tabs/tabs-content.ts [hunk 1]
```diff
@@ -1,23 +1,23 @@
 import 'Frontend/demo/init'; // hidden-source-line
 
-import { html, LitElement, customElement, internalProperty, css } from 'lit-element';
+import { html, LitElement, customElement, internalProperty } from 'lit-element';
 import '@vaadin/vaadin-tabs/vaadin-tabs';
+import '@vaadin/vaadin-ordered-layout/vaadin-vertical-layout';
+import '@vaadin/vaadin-lumo-styles/typography';
+import { applyTheme } from 'generated/theme';
```
Comment (sosa-vaadin) [inline frontend/demo/component/tabs/tabs-content.ts:7]:
```suggestion
import { applyTheme } from 'Frontend/generated/theme';
```

**Review by sosa-vaadin — CHANGES_REQUESTED** [review]
Fix an import that's breaking the netlify deployment at line 7 in `tabs-content.ts`. Everything else looks good.
The badges example works on my machine, I don't know if you fixed it and didn't update the comment on this PR but I don't see it failing.

File: frontend/demo/component/tabs/tabs-content.ts (hunk 1 above)
Comment (anezthes) [reply frontend/demo/component/tabs/tabs-content.ts:7]: Fixed, thanks! Great if the badges work now.

Comment (anezthes) [reply frontend/demo/component/tabs/tabs-content.ts:7]: Badges look good in the Java examples, but not in the TS ones:

Screenshot 2021-05-19 at 11 45 24

**Review by sosa-vaadin — APPROVED** [review]

**Pull request merged**